code/game/objects/items/devices/miragedrive.dm

```diff
@@ -14,26 +14,11 @@
 	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
 	item_state = "mdrive"
 	w_class = WEIGHT_CLASS_SMALL
-
 	COOLDOWN_DECLARE(last_dash)
 	COOLDOWN_DECLARE(last_attack)
 	var/obj/item/card/id/access_card
 	var/list/moving = list()
 
-/obj/item/mirage_drive/Initialize(mapload)
-	. = ..()
-	access_card = new /obj/item/card/id/captains_spare()
-
-/obj/item/mirage_drive/Destroy(force=FALSE)
-	QDEL_NULL(access_card)
-	moving = null
-	return ..()
-
-/obj/item/mirage_drive/examine(datum/source, mob/user, list/examine_list)
-	. = ..()
-	if(!COOLDOWN_FINISHED(src, last_dash))
-		. += span_notice("A digital display on it reads [COOLDOWN_TIMELEFT(src, last_dash)/10].")
-
```

Comment:
Whats all this about?

Reply:
I plan to take another look at some point, but i'll see if i can get a testmerge going

Reply:
auto merge bug, i'll fix

Reply:
first 2 procs are because you don't need ID card anymore to test for pathing

Reply:
fair enough, the examine is still needed to figure out the cooldown though